Gemfile

```diff
@@ -1,8 +1,9 @@
 source 'https://rubygems.org'
-ruby '2.2.2'
+ruby '2.2.3'
 
-gem 'rails', '4.1.6'
+gem 'rails', '~>4.2.4'
```

Comment:
It'd be nice to get up to Rails 5.0.0-beta1 eventually, but we have bigger :fish: to :fire: